utils/io/content/contentreader.go

```diff
@@ -0,0 +1,158 @@
+package content
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"sync"
+
+	"github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+type ContentReader struct {
+	// Response data file path.
+	filePath, arrayKey string
+	// The objects from the response data file are being pushed to the data channel.
```

Comment:
...response data file... --> ...source data file...

...being pushed to the... --> ...being pushed into the...